src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/RepositoryTestSuite.kt

```diff
@@ -15,7 +17,7 @@ import org.vaccineimpact.api.test_helpers.DatabaseCreationHelper
 // Keep these sorted alphabetically, for consistency
 @RunWith(Suite::class)
 @Suite.SuiteClasses(
-        BurdenEstimateRepositoryTests::class,
+        UploadBurdenEstimateTests::class,
         BurdenEstimateRepositoryTestsDeprecated::class,
```

Comment:
You'll want to add Retrieve... tests in here too

Reply:
Fixed - good catch :)